e2e-tests/web/test/specs/changeLanguage.e2e.ts

```diff
@@ -15,6 +15,7 @@ describe('change language', () => {
     await languageIcon.click()
 
     const languageSelector = await $("*[data-testid='headerActionItemDropDown']")
+    await $('=English')
```

Comment:
Can you explain why this makes the tests less flaky? :)

Reply:
My guess is that the `headerActionItemDropDown` (i.e. the box that contains all the languages that you can choose) sometimes is rendered but in a state where it's still not visible, so the test for the visibility would fail. But when awaiting one of its child elements to show up, that would mean that it has to be visible, so the test passes.